css/src/components/scroll.scss

```diff
@@ -1,3 +1,13 @@
+$scrollPaddingTop: var(--scroll-padding-top);
```

Comment:
component files should not contain any html non-class scoped rules. Can you help me understand this one? Should it remain in docs-ui somewhere? Or should it be placed in a new file in the core/ folder? 

If it is not something we're certain should be universally applicable, then I would suggest that we do not add it. It seems to me that something as specific as 17vh might be quite situational.

Reply:
removed